src/common/indexer/elastic/elastic.indexer.helper.ts

```diff
@@ -76,7 +76,15 @@ export class ElasticIndexerHelper {
   public buildCollectionRolesFilter(filter: CollectionFilter, address?: string): ElasticQuery {
     let elasticQuery = ElasticQuery.create();
     elasticQuery = elasticQuery.withMustNotExistCondition('identifier')
-      .withMustMultiShouldCondition([NftType.MetaESDT, NftType.NonFungibleESDT, NftType.SemiFungibleESDT], type => QueryType.Match('type', type));
+      .withMustMultiShouldCondition([
+        NftType.MetaESDT,
```

Comment:
you could have used `Object.values(NftType)` instead of stating each member of the enum. also future proof